recipes/testcontainers/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "testcontainers" %}
+{% set version = "4.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/testcontainers-{{ version }}.tar.gz
+  sha256: f4e87c3831991f9b4d5b6544d0f438fbfd140a74c834b13efb15a2f7c89833c4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.9,<4.0
+    - docker-py
+    - urllib3
+    - wrapt
+    - typing-extensions
+  run_constrained:
+    - python-arango >=7.8.0,<8.0.0
+    - azure-storage-blob >=12.19.0,<13.0.0
+    - clickhouse-driver *
+    - google-cloud-pubsub >=2
+    - google-cloud-datastore >=2
+    - influxdb *
+    - influxdb-client *
+    - python-kubernetes *
+    - pyyaml *
+    - python-keycloak *
+    - boto3 *
+    - minio *
+    - nats-py *
+    - pymongo *
+    - sqlalchemy *
+    - pymssql *
+    - pymysql *
+    - neo4j-python-driver *
+    - opensearch-py *
+    - oracledb *
+    - pika *
+    - redis-py *
+    - selenium *
+    - weaviate-client >=4.5.4,<5.0.0
+    - chromadb-client *
+    - qdrant-client *
+    - bcrypt *
```

Comment:
Please delete this as this will otherwise make coinstalling this package with something else really hard.

Reply:
Even the ones where a concrete version pinning is declared, e.g. `weaviate-client`?

Reply:
Even the ones with a concrete version pinning, e.g. weaviate-client?

Reply:
And does having `*` as constraint have an effect? I thought `run_constrained` meant that only WHEN the package is installed, the specific version (here `*`) is needed

Reply:
All of them please. This is a wrong implementation of optional dependencies in grayskull